backend/custom_models/migrations/0018_upload_pizza_test.py

```diff
@@ -0,0 +1,2093 @@
+# Generated by Django 2.1.7 on 2019-05-30 14:07
+# Edited by Francis Normand to upload the pizza test
+
+from django.db import migrations
+
+def upload_pizza_test(apps, schema_editor):
+    # get models
+    language = apps.get_model("custom_models", "Language")
+    item_type = apps.get_model("custom_models", "ItemType")
+    item = apps.get_model("custom_models", "Item")
+    item_text = apps.get_model("custom_models", "ItemText")
+    question_type = apps.get_model("custom_models", "QuestionType")
+    question = apps.get_model("custom_models", "Question")
+
+    # get db alias
+    db_alias = schema_editor.connection.alias
+
+    # lookup languages; do not use bulk_create since we need these objects later on
+    l_english = (
+        language.objects.using(db_alias)
+        .filter(ISO_Code_1="en", ISO_Code_2="en-ca")
+        .last()
+    )
+    l_french = (
+        language.objects.using(db_alias)
+        .filter(ISO_Code_1="fr", ISO_Code_2="fr-ca")
+        .last()
+    )
+    pizza_test_item_id = (
+        item_text.objects.using(db_alias)
+        .filter(text_detail="Pizza Test", language=l_english)

```

Comment:
As discussed with Caley offline, we need to shift from looking up via text detail to using the test name. I will be changing this for the eMiB globally in a PR in the near future. I'll add a comment here when I do so you can do the same thing for the pizza test

Reply:
See PR #446

Reply:
Updated in commit #https://github.com/code-for-canada/project-thundercat/pull/445/commits/1e786ad66593d49ddc46e3b745f54287b22f7ec8